docs/cas-server-documentation/installation/Ticket-Registry-Replication-Encryption.md

```diff
@@ -10,6 +10,7 @@ by encrypting and signing tickets:
 * [Hazelcast](Hazelcast-Ticket-Registry.html)
 * [Ehcache](Ehcache-Ticket-Registry.html)
 * [Ignite](Ignite-Ticket-Registry.html)
+* [CouchDb](CouchDb-Ticket-Registry.html)
```

Comment:
Also list here? https://apereo.github.io/cas/development/installation/Configuring-Ticketing-Components.html

Reply:
It is, in the NoSQL stuff.